gulpfile.js

```diff
@@ -41,7 +41,8 @@ gulp.task('cards', () => (
     for (const file of files) { // eslint-disable-line
       const f = fs.readFileSync(`./cards/${file}`, { encoding: 'utf8' });
       const parsed = frontmatter(f);
-      cards.push(buildCardObject(file, parsed.attributes, parsed.body));
+      const body = parsed.body.replace(/^#(?!#).+/m, ''); // remove titles from body
```

Comment:
I wanted to keep that because it makes sense in terms of an API implementation to provide the main H1 to any other services requesting the card.